operator/src/lib.rs

```diff
@@ -71,17 +71,17 @@ impl NifiState {
             .collect::<Vec<_>>();
         let mut mandatory_labels = BTreeMap::new();
 
-        mandatory_labels.insert(String::from(APP_COMPONENT_LABEL), Some(roles));
-        mandatory_labels.insert(String::from(APP_INSTANCE_LABEL), None);
+        mandatory_labels.insert(String::from(labels::APP_COMPONENT_LABEL), Some(roles));
+        mandatory_labels.insert(String::from(labels::APP_INSTANCE_LABEL), None);
         mandatory_labels
     }
 
-    async fn create_config_map(
+    async fn build_config_map(
```

Comment:
Can you add documentation - especially about the return type and that you're checking for existence etc.